Model/Api.php

```diff
@@ -1814,6 +1814,30 @@ public function kickoffParallelCampaigns(Contact $contact, &$campaignIds = [], $
             return $this;
         }
 
+        $ramRequiredPercent = intval($this->getIntegrationSetting('parallel_ram', 20));
+        if ($ramRequiredPercent > 0) {
+            try {
+                $memInfo = file_get_contents('/proc/meminfo');
+                preg_match('#MemTotal:[\s\t]+([\d]+)\s+kB#', $memInfo, $a);
+                if (isset($a[1])) {
+                    $memTotal = (int) $a[1];
+                    preg_match('#MemAvailable:[\s\t]+([\d]+)\s+kB#', $memInfo, $b);
+                    if (isset($b[1])) {
```

Comment:
Kind of nit picky, but I just don't like seeing $a $b variable names, I think it's fine in this case though.

Reply:
True, I just didn't want to continue using $matches since it's by ref. and I don't know the behavior of preg_match when the var already exists.